be/src/exec/pipeline/pipeline.h

```diff
@@ -21,11 +21,13 @@
 #include "exec/pipeline/pipeline_fwd.h"
 #include "exec/pipeline/source_operator.h"
 #include "gutil/strings/substitute.h"
-namespace starrocks::pipeline {
 
-class Pipeline;
-using PipelinePtr = std::shared_ptr<Pipeline>;
-using Pipelines = std::vector<PipelinePtr>;
+namespace starrocks {
+
+class RuntimeState;
```

Comment:
put it into the pipeline_fwd ?

Reply:
Maybe `pipeline_fwd` should put the forward declarations which other module need when they use pipeline module?